src/common/cockpitwebinject.c

```diff
@@ -174,6 +185,7 @@ cockpit_web_inject_new (const gchar *marker,
   self = g_object_new (COCKPIT_TYPE_WEB_INJECT, NULL);
   self->marker = g_bytes_new (marker, len);
   self->inject = g_bytes_ref (inject);
+  self->maximum = count;
```

Comment:
Note that this is also not where the actual object is created. It's created just before cockpit_web_inject_init().